lib/adapter.js

```diff
@@ -155,6 +159,7 @@ class MochaAdapter {
         if (err) {
             this.runner.lastError = err
         }
+        process.emit(event, message)
```

Comment:
Yes, exactly. I couldn't find any other way of keeping the runner aware of the current test, in order to report the current test on command/result/screenshot/other wdio events. Can you?